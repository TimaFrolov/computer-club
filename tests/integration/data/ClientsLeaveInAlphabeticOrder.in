1
01:00 02:00
7
01:15 1 b
01:20 1 a
01:25 1 c
01:30 1 d
