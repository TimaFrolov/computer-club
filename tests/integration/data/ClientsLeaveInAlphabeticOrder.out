01:00
01:15 1 b
01:20 1 a
01:25 1 c
01:30 1 d
02:00 11 a
02:00 11 b
02:00 11 c
02:00 11 d
02:00
1 0 00:00
